internal/confmapprovider/discovery/bundle/bundle.d/receivers/mongodb.discovery.yaml

```diff
@@ -42,6 +41,12 @@ mongodb:
               `--set splunk.discovery.receivers.mongodb.config.password="<password>"` or
               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_username="<username>"` and
               `SPLUNK_DISCOVERY_RECEIVERS_mongodb_CONFIG_password="<password>"` environment variables.
+      - status: partial
+        regexp: 'SSL handshake received but server is started without SSL support'
+        log_record:
+          append_pattern: true
+          body: >-
+            Make sure the target database has SSL enabled.
```

Comment:
Can we give more specific instructions?

Reply:
Tested for both SSL enabled/ without SSL enabled. All the edge cases work now.

Reply:
After giving it a bit of thought I have decided to remove this partial failure altogether as it may increase confusion.